Lesson3/ViewController.swift

```diff
@@ -1,95 +1,71 @@
-//
-//  ViewController.swift
-//  Lesson3
-//
-//  Created by i.ikhsanov on 21.09.2021.
-//
-
 import UIKit
 
+
 class ViewController: UIViewController {
-    
     @IBOutlet weak var tableView: UITableView!
-    
-    var students: [Student] = [Student(name: "Alex", groupNumber: "11-108"),
-                               .init(name: "Mikal", groupNumber: "11-205"),
-                               .init(name: "Bulat", groupNumber: "12-304"),
-                               .init(name: "Marat", groupNumber: "11-103")]
+    @IBOutlet weak var IMAGE: UIImageView!
```

Comment:
```suggestion
    @IBOutlet weak var что-тоImageView: UIImageView!
```
что за стиль? я так никогда не делал -0,5